packages/core/src/fields/types/relationship/views/RelationshipSelect.tsx

```diff
@@ -107,6 +112,31 @@ export function useFilter (value: string, list: ListMeta, searchFields: string[]
       })
     }
 
+    for (const { field, refSearchFields, many } of relationsSearchFields) {
+      conditions.push(
+        ...refSearchFields.map((refSearchField) =>
+          many
+            ? {
+                [field]: {
+                  some: {
+                    [refSearchField]: {
+                      contains: trimmedSearch,
```

Comment:
these fields having `contains` only works with the `text` field. This should be documented.

Reply:
@pnodet can you update the documentation to mention that fields as listed in `searchFields` need to support the `contains` filter?

Reply:
Actually @gautamsi, we should add that as a different pull request, this applies to `searchFields` generally